articles/quickstart/webapp/aspnet-owin/_includes/_login.md

```diff
@@ -210,7 +205,9 @@ You will also need to configure the OpenID Connect middleware to add the ID Toke
 
 Update the OpenID Connect middleware registration in your `Startup` class as follows:
 
-1. Set the `ResponseType` to `OpenIdConnectResponseType.CodeIdTokenToken`. This will inform the OpenID Connect middleware to extract the Access Token and store it in the `ProtocolMessage`.
+1. Set the `ResponseType` to `OpenIdConnectResponseType.Code`. This will inform the OpenID Connect middleware to extract the Access Token and store it in the `ProtocolMessage`.
```

Comment:
You want `id_token` for Implicit Flow with Form Post

Reply:
Sorry, the PR title is misleading. This PR is defaulting to Implicit, but still guides to use Code in the section `Obtain an Access Token for calling an API`.